source/whats-new.txt

```diff
@@ -0,0 +1,168 @@
+.. _cpp-whats-new:
+
+==========
+What's New
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: update, new feature, deprecation, upgrade, driver
+
+Overview
+--------
+
+Learn what's new in:
+
+* :ref:`Version 3.10 <version-3.10>`
+* :ref:`Version 3.9 <version-3.9>`
+
+Breaking Change Types
+~~~~~~~~~~~~~~~~~~~~~
+
+Breaking changes introduced by new versions of the {+driver-short+} belong to one of the
+following groups:
+
+- **API** changes concern the API of the {+driver-short+} libraries. These changes are
+  accompanied by an API major version bump. You might need to edit C++ source code
+  in your project to address these changes.
+
+  .. example::
+
+     - Removed function ``func(A)``. Use ``func(A, B)`` instead.
+     - Changed redeclaration of ``bsoncxx::foo`` from ``v1::foo`` to ``v2::foo``. Qualify
+       ``bsoncxx::foo`` as ``bsoncxx::v1::foo`` to continue using the old interface.
+
+- **ABI** changes concern the ABI of the {+driver-short+} libraries. These changes are
+  accompanied by an ABI version bump, and may also be accompanied by an API major version
+  bump. You might need to rebuild your project or edit your C++ source code to address
+  these changes.
+
+  .. example::
+
+     - Removed exported symbol for ``v1::func(A)``. Use ``v2::func(A)`` instead.
+
+- **Build System** changes concern the build system or packaging of the {+driver-short+}
```

Comment:
S: I don't think "system" needs to be capitalized, since "build system" is lowercase in all other mentions
```suggestion
- **Build system** changes concern the build system or packaging of the {+driver-short+}
```

Reply:
I had it like this originally, but changed it to uppercase S to align with the way it's formatted in the admonition later on. I prefer lowercase too, though